apps/vs-code-react/src/webviewCommunication.tsx

```diff
@@ -1,46 +1,153 @@
-import type {
-  InjectValuesMessage,
-  UpdateAccessTokenMessage,
-  UpdateExportPathMessage,
-  AddStatusMessage,
-  SetFinalStatusMessage,
+import {
+  type UpdateAccessTokenMessage,
+  type UpdateExportPathMessage,
+  type AddStatusMessage,
+  type SetFinalStatusMessage,
+  type FetchSchemaMessage,
+  type LoadDataMapMessage,
+  type ShowAvailableSchemasMessage,
+  type GetAvailableCustomXsltPathsMessage,
+  type SetXsltDataMessage,
+  type SetRuntimePortMessage,
+  type GetConfigurationSettingMessage,
+  type InjectValuesMessage,
+  type UpdatePanelMetadataMessage,
+  type CompleteFileSystemConnectionMessage,
+  type ReceiveCallbackMessage,
+  ProjectName,
 } from './run-service';
-import { initialize, updateAccessToken, updateTargetDirectory, addStatus, setFinalStatus } from './state/WorkflowSlice';
-import type { AppDispatch } from './state/store';
+import {
+  changeCustomXsltPathList,
+  changeDataMapMetadata,
+  changeMapDefinition,
+  changeRuntimePort,
+  changeSchemaList,
+  changeSourceSchemaFilename,
+  changeTargetSchemaFilename,
+  changeUseExpandedFunctionCards,
+  changeXsltContent,
+  changeXsltFilename,
+} from './state/DataMapSlice';
+import { initializeDesigner, updateCallbackUrl, updateFileSystemConnection, updatePanelMetadata } from './state/DesignerSlice';
+import type { InitializePayload } from './state/WorkflowSlice';
+import {
+  initialize as initializeWorkflow,
```

Comment:
Lets change it to `initializeWorkflow` and keep the initialize to the project, therefore whenever we imported it we need to import it with an alias